recipes-ros/mavlink/mavlink_2016.5.20.bb

```diff
@@ -0,0 +1,22 @@
+#
+# MAVLink message marshaling library.
+# This package provides C-headers and pymavlink library
+# for both 1.0 and 2.0 versions of protocol.
+#
+
+DESCRIPTION = "MAVLink message marshaling library"
+DEPENDS = ""
+LICENSE = "LGPLv3"
+LIC_FILES_CHKSUM = "file://COPYING;md5=54ad3cbe91bebcf6b1823970ff1fb97f"
+
+SRCBRANCH = "release/indigo/mavlink"
+SRCREV = "977f7d41d0a4e0afbe67cdb61f0d69eff2c2fd81"
+SRC_URI = "git://github.com/mavlink/mavlink-gbp-release.git;branch=${SRCBRANCH}"
+
+S = "${WORKDIR}/git"
+
+inherit catkin
+
+
```

Comment:
Four empty lines at the end seems like a lot of empty space here. I prefer a final line break and no empty line at the end, so that `cat mavlink_2016.5.20.bb` gives a short readable output.